cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,60 @@
 /// <reference types='cypress' />
 
+const { generateUser } = require('../support/generate');
+
 describe('Student Registration page', () => {
   before(() => {
-    
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
 
-  it('', () => {
+  it('should fill all the fields and submit the form', () => {
+    const user = generateUser();
```

Comment:
where I can see generateUser function?

Reply:
it was not added for some reason, committed once more